### PR TITLE
1236-allow-adding-people-responders

### LINKED ISSUE #1236: Allow adding people as responders
+ assign yourself shorcut

### PR DESCRIPTION
**Pull request by mainawycliffe**
Closes #1236

### CODE REVIEW

**Comment by moshloop** [comment]
@mainawycliffe  for the popup can we do:

```
Person: {dropdown}
     OR
Team: {Dropdown} 
```

Rather than a switcher.

Also the new responder isn't immediately added to the panel, nor do I seem them on the incidents list page

**Comment by mainawycliffe** [comment]
@moshloop like this:

image

**Comment by moshloop** [comment]
@mainawycliffe  - Without the inline Label and with an `OR` text between them 
The modal title also needs to be changed

**Comment by moshloop** [comment]
@mainawycliffe    can we do this vertically rather than horizontal with the labels on the left instead of the top ?

**Comment by mainawycliffe** [comment]
@moshloop sure, in terms of the responder column, I am working on displaying them, might need to do some refactoring around.

**Comment by moshloop** [comment]
@mainawycliffe - this is better

- Can we move the add responders to after the list of responders
- The avatar icon doesn't seem consistent between incident changelog (what it should be) and the responders list in both details and the incident list

File: src/api/services/responder.ts [hunk 1]
```diff
@@ -1,18 +1,103 @@
 import { IncidentCommander } from "../axios";
 import { resolve } from "../resolve";
+import { Team } from "./teams";
+import { User } from "./users";
+
+/**
```
Comment (moshloop) [inline src/api/services/responder.ts:6]: remove this comment

**Comment by moshloop** [comment]
I think we should also be using the incident_summary view: https://github.com/flanksource/duty/blob/main/views/009_incident_views.sql#L56

**Comment by moshloop** [comment]
@mainawycliffe Can we make the avatars rounded and sm by default so they are consistent across the board?

**Comment by mainawycliffe** [comment]
@moshloop like this:

image

**Pull request merged**